### DISCUSSION TITLE
Change encryption Key of credentials-config.json

### DISCUSSION THREAD

**Discussion by OpsMB** [dbeaver/dbeaver#12662] [DBeaver Development]
Hello! I tried to build a version of DBeaver to change the encryption key of the credentials-config.json file.
The new build is functional. The problem is that I will have to change this key after each DBeaver update.

Is it possible to have an option on DBeaver to specify this key on the installation for exemple or on "Security" menu.

I searched on the forum and on google and I don't find anything about this subject, I'm sorry if this question was already ask on the forum. 

Thank you

MB

**Comment by ShadelessFox** [comment]
Hello @OpsMB,

Thanks for the suggestion. I will create a issue for your feature request.

Update: #12717

**Reply by OpsMB** [reply]
Hello @ShadelessFox !

Thank you for the reply.

MB
